poc/extensions/src/extension_core.rs

```diff
@@ -0,0 +1,74 @@
+use super::Dispatchable;
+use super::ExtensionError;
+use super::ExtensionTypeId;
+use parity_scale_codec::{Decode, Encode};
+// SDK codes
+pub trait ExtensionCore: Dispatchable + TryFrom<(u32, Vec<u8>)> {
```

Comment:
`TryFrom<(u32, Vec<u8>)` is not correct. It should be something like `TryFrom<(Dispatchable::MethodIndex, Dispatchable::MethodName)>`
(you need to figure out the right syntax. maybe with a whare clause(

Reply:
and should have a `trait Extension: Dispatchable + TryFrom<X, Y>` and have other extensions conforming it